app/src/org/commcare/preferences/DeveloperPreferences.java

```diff
@@ -205,6 +207,11 @@ public static boolean shouldFireTriggersOnSave() {
         return properties.getString(FIRE_TRIGGERS_ON_SAVE, CommCarePreferences.YES).equals(CommCarePreferences.YES);
     }
 
+    public static boolean shouldAnimateFormSubmitButton() {
+        SharedPreferences properties = CommCareApplication._().getCurrentApp().getAppPreferences();
+        return properties.getString(ANIMATE_FORM_SUBMIT_BUTTON, CommCarePreferences.YES).equals(CommCarePreferences.YES);
```

Comment:
@phillipm this should be `properties.getString(ANIMATE_FORM_SUBMIT_BUTTON, CommCarePreferences.NO)` in order to make it default to off